File: app/src/main/java/com/base/base_app/data/model/ticket/Role.java
package com.base.base_app.data.model.ticket;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Role {

    @SerializedName("Id")
    @Expose
    private String id;
    @SerializedName("Description")
    @Expose
    private String description;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }
}

File: app/src/main/java/com/kasra/atency/data/model/ticket/Role.kt
package com.kasra.atency.data.model.ticket

import com.google.gson.annotations.SerializedName
import com.google.gson.annotations.Expose

data class Role (
    @SerializedName("Id")
    @Expose
    var id: String? = null,

    @SerializedName("Description")
    @Expose
    var description: String? = null
)